examples/defer/defer.go

```diff
@@ -40,5 +40,9 @@ func writeFile(f *os.File) {
 
 func closeFile(f *os.File) {
 	fmt.Println("closing")
-	f.Close()
+	err := f.Close()
+	if err != nil {
```

Comment:
I would add a comment here so it'd show properly in the left pane - saying something like 

"It is important to check for errors when closing a file, even in a deferred function."

Reply:
done